usefulUtilities/hiFiCalendar/resources/test.html

```diff
@@ -0,0 +1,152 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <title>Google OAuth API Quickstart</title>
+  <meta charset="utf-8" />
+</head>
+<body>
+    <p>Google OAuth API Quickstart</p>
+    <pre id="content" style="white-space: pre-wrap;"></pre>
+    <script type="text/javascript">
+        // Load the API's client and auth2 modules.
+        // Call the initClient function after the modules load.
+        var API_KEY = '';
+        var CLIENT_ID = '';
+        var CLIENT_SECRET = '';
+        var SCOPE = 'https://www.googleapis.com/auth/calendar.events.readonly';
+        var authCode = false;
+        var GoogleAuth;
+        function handleClientLoad() {
+            gapi.load('client:auth2', initClient);
+        }
+    
+        function initClient() {
+            var discoveryUrl = 'https://www.googleapis.com/discovery/v1/apis/calendar/v3/rest';
+            gapi.client.init({
+                'apiKey': API_KEY,
+                'discoveryDocs': [discoveryUrl],
+                'clientId': CLIENT_ID,
+                'scope': SCOPE
+            }).then(function () {
+                GoogleAuth = gapi.auth2.getAuthInstance();
+        
+                // Listen for sign-in state changes.
+                GoogleAuth.isSignedIn.listen(updateSigninStatus);
+                GoogleAuth.grantOfflineAccess().then(function (response) {
+                    authCode = response.code;
+                    var zone = new Date().toLocaleTimeString('en-us',{timeZoneName:'short'}).split(' ')[2];
```

Comment:
this seems very fragile, is there no other way to get this data?

Reply:
Are you asking about getting the timezone name? this is the only way I found.